diff --git a/dist/vitest-plugin/test-utils.js b/dist/vitest-plugin/test-utils.js
index 40c323d79fe71610148897e983ed01f134e28f88..b01010733db7328b9cec443db736e6f8a057abd6 100644
--- a/dist/vitest-plugin/test-utils.js
+++ b/dist/vitest-plugin/test-utils.js
@@ -51,7 +51,7 @@ var DEFAULT_VIEWPORT_DIMENSIONS = {
 }, setViewport = async (parameters = {}, globals = {}) => {
   let defaultViewport, viewportsParam = parameters.viewport ?? {}, viewportsGlobal = globals.viewport ?? {}, isDisabled = viewportsParam.disable || viewportsParam.disabled;
   viewportsGlobal.value && !isDisabled ? defaultViewport = viewportsGlobal.value : isDisabled || (defaultViewport = viewportsParam.defaultViewport);
-  let { page } = await import("@vitest/browser/context").catch(() => ({
+  let { page } = await import("vitest/browser").catch(() => ({
     page: null
   }));
   if (!page || !globalThis.__vitest_browser__)
